src/main/java/org/akhq/modules/KafkaModule.java

```diff
@@ -126,15 +128,7 @@ public AdminClient getAdminClient(String clusterId) throws InvalidClusterExcepti
     }
 
     public KafkaConsumer<byte[], byte[]> getConsumer(String clusterId) throws InvalidClusterException {
-        if (!this.clusterExists(clusterId)) {
-            throw new InvalidClusterException("Invalid cluster '" + clusterId + "'");
```

Comment:
This test must be keep I think

Reply:
Hello, any news ?
According my comment, this check seems me redondant,
do you still waiting for this changes ?